server/index.js

```diff
@@ -65,6 +73,8 @@ const bootApp = () => {
     await createApp()
     await startListening()
   })
+
+  autoCleanDB()
```

Comment:
Need to `await` here since autoCleanDB is async. Besides that looks good

Reply:
so the pattern is await absolutely everything possible even if nothing else depends on it? I thought we only awaited if things depended on each other

Reply:
Yeah good point. Let's leave it